scripts/get-version

```diff
@@ -1,7 +1,12 @@
 #!/bin/bash
 set -o nounset -o errexit -o pipefail
-SCRIPT_DIR="$( cd "$( dirname "$0" )" && pwd )"
-COMMITISH=${1:-HEAD}
+
+if [ $# -eq 0 ]; then
+    echo "No arguments provided. Pass appropriate git commit-ish value get version of (i.e. 'HEAD')"
```

Comment:
Why change this?

Reply:
Previously the command supported passing no args, or one arg, with the arg passed in being assumed to be hte 'commit-ish' flag to pass to git commands.  I added support for a second arg that allows the caller to indicate if they want to embed the feature-tag into the version-string produced.  However, this produced a lot of extra complexity as the the command now needs to handle 0, 1, or 2 args, and needed to understand in the 1-arg case if this was the commit-ish arg, or hte `embed` arg.    So i simplified things overall by making the commit-ish arg mandatory.  So all callers always pass it in, and callers can optionally pass in the `embed` arg as well.